src/passage/Passage.sol

```diff
@@ -125,7 +128,8 @@ contract Passage is PassagePermit2 {
     function withdraw(address token, address recipient, uint256 amount) external {
```

Comment:
should this be non-reentrant?

Reply:
per description, since it's only callable by the tokenAdmin i don't think we need to worry about reentrancy